src/seedu/addressbook/commands/Command.java

```diff
@@ -15,6 +15,7 @@
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
     private int targetIndex = -1;
+    public boolean MUTATED_DATA = true;
```

Comment:
If `MUTATED_DATA` is a constant,  then it should be defined as `static final`. However, it's unnecessary to create a constant/variable for this case.